apps/aeoracle/test/aeoracle_SUITE.erl

```diff
@@ -133,15 +135,33 @@ register_oracle_negative_dynamic_fee(_Cfg) ->
 
     %% Test minimum fee for increasing TTL.
```

Comment:
I see these tests may be made more maintainable and precise by retrieving the fraction returned by `aec_governance:state_gas_per_block`, choosing numbers for preventing serialization to cross boundary hence changing required fee. Though ok for now.